Source/Components/Touchpoints/Types/FlexCoverCarousel/Card/MLBusinessFlexCoverCarouselItemView.swift

```diff
@@ -215,8 +223,12 @@ public class MLBusinessFlexCoverCarouselItemView: UIView {
             }
         }
         
-        if let colorString = item.backgroundColor {
-            mainCardContainerView.backgroundColor =  colorString.hexaToUIColor()
+        if let colorString = item.backgroundColor, colorString != "" {
+            mainCardDefaultHeightConstraint.constant = MLBusinessFlexCoverCarouselItemView.containerHeight
```

Comment:
y aca tengo una duda, para que le volves a aplicar el valor no se lo habias hecho aca?
`private lazy var mainCardDefaultHeightConstraint = mainCardContainerView.heightAnchor.constraint(equalToConstant:MLBusinessFlexCoverCarouselItemView.containerHeight)`

Reply:
Si, en realidad tenía la duda de ponerlo acá o en el clear. Porque la idea es que la altura del sólido cambie cuando viene el color de BE. Te parece que lo lleve a ese método, así queda mas claro?